lib/codecs/src/decoding/format/native_json.rs

```diff
@@ -132,8 +132,8 @@ mod test {
 
         let events = deserializer.parse(input, LogNamespace::Legacy).unwrap();
 
-        let event1 = Event::try_from(json1).unwrap();
-        let event2 = Event::try_from(json2).unwrap();
+        let event1 = Event::from_json_value(json1, LogNamespace::Legacy).unwrap();
```

Comment:
Unrelated, but this is making me wonder: is data passed through correctly from Vector sink to Vector source when using log namespacing on either or both sides?

Reply:
The Vector sink/source uses protobuf to send the events. The protobuf encodes the raw `Value` for the event and metadata, so it should keep it in the correct format regardless of the log namespacing used. (It used to store only only a map for the event, assuming it was an object, but that was changed a while ago).